lib/VMwareWebService/broker_timeout.rb

```diff
@@ -0,0 +1,64 @@
+module Timeout
```

Comment:
Please insert comment #hack #todo comment indicating that we're re-implementing stdlib's Timeout.timeout for the broker, why (link to https://bugzilla.redhat.com/show_bug.cgi?id=1207018), where stdlib's version is broken, and how this implementation improves the situation.  We should also say something about re-testing this when we upgrade rubies as we either need to reimplement it for newer rubies or use an alternative solution.

We should really link to the BZ issue. I can help rewrite the commit messages with a link to https://bugzilla.redhat.com/show_bug.cgi?id=1207018.